CHANGELOG.md

```diff
@@ -13,7 +13,7 @@
 * Fix optimizations bypassing our custom getter and setter from within a class (Issue [#375](https://github.com/realm/realm-kotlin/issues/375)).
 
 ### Compatibility
-* This release is compatible with Kotlin 1.5.10 and Coroutines 1.5.0.
+* This release is compatible with Kotlin 1.5.21 and Coroutines 1.5.0.
 
 ### Internal
```

Comment:
We should probably also mention it in the internal section, just because it is easy to miss under compatibility